MobileCenterUpdates/MobileCenterUpdates/MSUpdates.m

```diff
@@ -67,20 +82,71 @@ + (instancetype)sharedInstance {
   static id sharedInstance = nil;
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
-      sharedInstance = [[self alloc] init];
+    sharedInstance = [[self alloc] init];
   });
   return sharedInstance;
 }
 
-- (void)startWithLogManager:(id <MSLogManager>)logManager appSecret:(NSString *)appSecret {
+- (void)startWithLogManager:(id<MSLogManager>)logManager appSecret:(NSString *)appSecret {
   [super startWithLogManager:logManager appSecret:appSecret];
+
+  // TODO: Hook up with update token getter later.
+  NSString *updateToken = @"temporary-token";
+  self.sender =
+      [[MSDistributionSender alloc] initWithBaseUrl:self.updateUrl
+                                            apiPath:[NSString stringWithFormat:kMSUpdatesApiPathFormat, appSecret]
+                                            // TODO: Update token in header should be in format of "Bearer {JWT token}"
+                                            headers:@{
+                                              kMSUpdatesHeaderApiToken : updateToken
+                                            }
+                                       queryStrings:nil
+                                       reachability:[MS_Reachability reachabilityForInternetConnection]
+                                     retryIntervals:@[ @(10) ]];
+
   MSLogVerbose([MSUpdates logTag], @"Started Updates service.");
+
```

Comment:
This is an intended line break. I want to separate the code blocks in the method for starting logic and post-starting process. The line break is necessary for semantic.